packages/govuk-frontend/src/govuk/components/checkboxes/_index.scss

```diff
@@ -236,8 +217,8 @@
     //
     // Reduce the size of the check mark and re-align within the checkbox
     .govuk-checkboxes__label::after {
-      top: 15px;
-      left: 6px;
+      top: 17px;
+      left: (16px - $input-offset);
```

Comment:
Are we able to explain these offsets in relation to "container size" minus "checkbox width" etc?

Reply:
I'll write something up. Basically it's not feasible to automate the positioning of the check mark because it's too complex a shape, but we can determine a few things like it's left position based on the small input offset.